tools.yaml

```diff
@@ -34,3 +34,321 @@ batman.r-forge.r-project.org:
   license:
     kind: Boost Software License Version 1.0
     url: "https://r-forge.r-project.org/scm/viewvc.php/*checkout*/pkg/batman/LICENSE_1_0.txt?root=batman"
+github.com/dgrapov/CTSgetR:
+  name: CTSgetR
+  description: |
+    "R interface to Chemical Translation Service (CTS)"
+  url: https://github.com/dgrapov/CTSgetR
+  authors: 
+    - name: Dmitry Grapov
+      email: dgrapov@gmail.com
+  license:
+    - kind: MIT
+    - url: https://opensource.org/licenses/MIT
+  tags:
+    - R
+
+stat.ethz.ch/R-manual/R-patched/library/stats/html/hclust.html:
```

Comment:
For the record, these keys are meant to be unique identifiers, but we also use it to generate the IRI in VIVO. This is an unfortunately long ID which we may need to change in the future, but leave it for now.